ui/src/components/select/QSelect.json

```
@@ -716,13 +716,9 @@
 
     "getOptionIndex": {
       "desc": "Gets current focused option index from menu; It's -1 if no option is focused",
-      "params": {
-        "index": {
-          "type": "Number",
-          "required": true,
-          "desc": "Index of option from menu; -1 if no option is focused",
-          "examples": [ 10 ]
-        }
+      "returns": {
+        "type": "Number",
+        "desc": "Index of option from menu; It's -1 if no option is focused"
```

```suggestion
        "desc": "Index of option from the menu; It's -1 if no option is focused",
        "examples": [ -1, 0, 10 ]
```

`examples` prop is required, you can see https://github.com/quasarframework/quasar/pull/12499#issuecomment-1043259518 for references and pointers about contributing and the build system.

---

Thanks for pointing this out, I hadn't set up the build to validate, sorry for that! I've added the return examples, but kept the return description wording the same since it matches setOptionIndex and moveOptionSelection from the same file.